### PR TITLE
Bug 1629647 - Create remote settings collection for Messaging Experiments

### PR DESCRIPTION
**Pull request by piatra**
The message is currently just a placeholder

### CODE REVIEW

File: messaging-experiments.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+- id: aw_pull_factor_privacy_1
```
Comment (ncloudioj) [inline messaging-experiments.yaml:1]: nit: shall we consider using capital case here?

Comment (k88hudson) [reply messaging-experiments.yaml:1]: good question, I think we should go with whatever is standard in normandy right now (looks like lowercase)

File: messaging-experiments.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    branches:
+      - slug: control
+        ratio: 1
+        value: {}
+      - slug: variant_1
+        ratio: 1
+        value:
+          title: Welcome to Firefox. Fast, safe, private.
+          cards:
+            - PERSONAL_DATA_PROMISE_CARD
```
Comment (k88hudson) [inline messaging-experiments.yaml:16]: These aren't real card values, can you just remove the `card` property in both branches for now?

File: messaging-experiments.yaml (hunk 1 above)
Comment (k88hudson) [reply messaging-experiments.yaml:1]: I take back my previous comment, this can be uppercase (or whatever we want)

Comment (ncloudioj) [reply messaging-experiments.yaml:1]: Yeah, I think Normandy's convention is using lowercase for slugs and branches, not sure about the recipe id. If it's up to us to decide,  uppercase would be easier to catch and manage.

**Comment by piatra** [comment]
To be deplyed once https://bugzilla.mozilla.org/show_bug.cgi?id=1630324 is fixed.

**Pull request merged**